docs/setting-up/server/docker.md

```diff
@@ -273,7 +273,7 @@ Use the following Docker container environment variables (with `-e var=value`) t
 | `METRICS_RESOLUTION_HR`    | High metrics resolution (same as above).
 | `METRICS_RESOLUTION_MR`    | Medium metrics resolution in seconds.
 | `METRICS_RESOLUTION_LR`    | Low metrics resolution in seconds.
-| `DATA_RETENTION`           | How many days to keep time-series data in ClickHouse.
+| `DATA_RETENTION`           | How many days to keep time-series data in ClickHouse.<br />**N.B.** this must be set in in a format supported by time.ParseDuration and represent a complete number of days.
```

Comment:
```suggestion
| `DATA_RETENTION`           | How many days to keep time-series data in ClickHouse.<br />**N.B.** this must be set in a format supported by `time.ParseDuration` and represent the complete number of days.
```

Reply:
@cezmunsta  Could we provide some examples for better clarity

Reply:
@rasika-chivate why `the` instead of `a`? It is representing _a_ complete number of days i.e. integer

Reply:
> Could we provide some examples for better clarity

I would argue that adding examples makes it worse :) The valid units are:
> Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".

The code should make it easy for the user and transform it to those, this is just to clarify that you can't use what makes sense :)